tools/devtool

```diff
@@ -601,6 +601,15 @@ cmd_test() {
     # We need to run a privileged container to get that kind of access.
     env |grep -P "^(AWS_EMF_|BUILDKITE_)" > env.list
 
+    # Disable turbo boost. Some of our tests are performance tests, and we want minimum variability wrt processor frequency
+    # See also https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/processor_state_control.html
+    sudo -n sh - c "echo 1 > /sys/devices/system/cpu/intel_pstate/no_turbo" 2>/dev/null
```

Comment:
Maybe save the value so we can restore it later

```sh
local saved_no_turbo=$(/sys/devices/system/cpu/intel_pstate/no_turbo
echo 1 | sudo tee /sys/devices/system/cpu/intel_pstate/no_turbo &>/dev/null
```

Reply:
Should we do the same thing for scaling_governor?

Reply:
That might be a bit more complicated because different CPUs might have different configs. From my testing on EC2 instances and local machines, "performance" seems to be the default here anyway, so not sure if its worth it? Maybe only doing this if the test is running in the Infra would be better, if there's an easy way to check for this in the bash script (maybe presence of some buildkite env var)?

Reply:
> Maybe only doing this if the test is running in the Infra would be better, if there's an easy way to check for this in the bash script (maybe presence of some buildkite env var)?

I agree with applying this only on our infra. `BUILDKITE="true"` seems to fit it.

https://buildkite.com/firecracker/firecracker-pr/builds/4268#018902ee-1b3c-485c-aae9-72e09809e719
> BUILDKITE="true"

Reply:
Ahh, amazing, I'll use that then!

Reply:
I thought about that, but as long as we also have perf tests in the normal CI, having it enabled always made sense to me